.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,13 +9,10 @@
 ### Preview link
 <!-- Impacted pages preview links-->
 
-<!-- This is the base preview link. This currently only works if you are in the Datadog organization and working off of a branch - it will not work with a fork. 
+<!-- This is the base preview link. This currently only works if you are in the Datadog organization and working off of a branch - it will not work with a fork.
 
 Replace the branch name and add the complete path:
 https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>
-
-For example, for branch "lucas/update-dotnet-tracing" that updates the docs in path "https://docs.datadoghq.com/tracing/languages/dotnet/", this is the preview link:
-https://docs-staging.datadoghq.com/lucas/update-dotnet-tracing/tracing/languages/dotnet/
 -->
```

Comment:
```suggestion

```

Reply:
we need this one to close the comment above